app/src/main/res/layout/profile_edit_activity.xml

```diff
@@ -209,12 +209,13 @@
             android:id="@+id/profile_edit_name"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
+            android:layout_marginEnd="150dp"
             android:fontFamily="sans-serif-medium"
             android:text="@{viewModel.profile.name}"
             android:textColor="@color/oppiaPrimaryText"
             android:textSize="20sp"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent" />
+            app:layout_constraintTop_toTopOf="parent"/>
```

Comment:
Please add this space

Reply:
> Please add this space

Ok I will add the space. I had a doubt, If I make a new commit now to my PR branch, will it be included in this PR or I'll have to make a new PR?

Reply:
No u just need to push to your branch it'll be automatically added in the PR